File: 2160. Minimum Sum of Four Digit Number After Splitting Digits/solution.py
class Solution(object):
  def minimumSum(self, num):
    """
    :type num: int
    :rtype: int
    """
    stringNum = str(num)
    sortedNum = sorted(stringNum)
    return int(sortedNum[0] + sortedNum[2]) + int(sortedNum[1] + sortedNum[3])

File: 2160. Minimum Sum of Four Digit Number After Splitting Digits/solution.js
/**
 * @param {number} num
 * @return {number}
 */
 var minimumSum = function(num) {
  let stringNum = num.toString();
  let sortedNum = stringNum.split('').sort().join('');
  return parseInt(sortedNum[0] + sortedNum[2]) + parseInt(sortedNum[1] + sortedNum[3]);
};